.circleci/config.yml

```diff
@@ -467,7 +467,8 @@ jobs:
       - run:
           name: Build Carthage
           command: |
-            carthage build --no-skip-current | tee buildout.txt
+            # Build Carthage via this script to workaround lipo error due to multiple architectures in same file
+            . ./CircleciScripts/carthage-build.sh --no-skip-current --platform iOS | tee buildout.txt
```

Comment:
I'm probably wrong, but this seems like this could just be:

```
./CircleciScripts/carthage-build.sh --no-skip-current --platform iOS | tee buildout.txt
```
??

Reply:
As long as the execute bit remains set, yes. That said, we've had pipeline errors before because a script wasn't set to execute, so I usually do a leading `.` to be defensive

Reply:
**Update** See below--executing with the dot-command causes the cleanup traps to not be executed. I'll change this to `bash` instead, to explicitly invoke a new subprocess and assign the traps to it